ci/config.rb

```diff
@@ -1,7 +1,8 @@
 $pipeline.pool('6.0-NSXV') do |pool|
   pool.params = {
     RSPEC_FLAGS: [
-      '--tag ~nsx_transformers',
+      '--tag ~nvds',
+      '--tag ~cvds',
```

Comment:
We can run `vsphere_networking` here too?

Reply:
Hmm it wasn't running before. I'm incline to say let's leave enabling `vsphere_networking` here for later?